stack/rest/src/test/java/org/apache/usergrid/rest/filters/ContentTypeResourceIT.java

```diff
@@ -105,106 +75,84 @@ public void correctHeaders() throws Exception {
-
-        assertEquals( 1, headers.length );
-
-        assertEquals( MediaType.APPLICATION_JSON, headers[0].getValue() );
     }
 
 
-    /** Tests that application/x-www-url-form-encoded works correctly */
+    /**
+     * Tests that application/x-www-url-form-encoded works correctly
+     */
     @Test
     public void formEncodedContentType() throws Exception {
 
-        List<NameValuePair> pairs = new ArrayList<NameValuePair>();
 
-        pairs.add( new BasicNameValuePair( "organization", "formContentOrg" ) );
-        pairs.add( new BasicNameValuePair( "username", "formContentOrg" ) );
-        pairs.add( new BasicNameValuePair( "name", "Test User" ) );
-        pairs.add( new BasicNameValuePair( "email", UUIDUtils.newTimeUUID() + "@usergrid.org" ) );
-        pairs.add( new BasicNameValuePair( "password", "foobar" ) );
+        Form payload = new Form();
+        payload.add("access_token", getAdminToken().toString());
+        payload.add("organization", "formContentOrg");
+        payload.add("username", "formContentOrg");
+        payload.add("name", "Test User");
+        payload.add("email", UUIDUtils.newTimeUUID() + "@usergrid.org");
+        payload.add("password", "foobar");
 
-        UrlEncodedFormEntity entity = new UrlEncodedFormEntity( pairs, "UTF-8" );
 
-        DefaultHttpClient client = new DefaultHttpClient();
+        ClientResponse clientResponse = app().users().getResource(false)
+            .accept(MediaType.APPLICATION_JSON)
+            .type(MediaType.APPLICATION_FORM_URLENCODED)
+            .post(ClientResponse.class, payload);
 
-        HttpHost host = new HttpHost( super.getBaseURI().getHost(), super.getBaseURI().getPort() );
+        assertEquals(200, clientResponse.getStatus());
 
-        HttpPost post = new HttpPost( "/management/orgs" );
-        post.setEntity( entity );
-        // post.setHeader(HttpHeaders.AUTHORIZATION, "Bearer " + context.getActiveUser().getToken());
+        MultivaluedMap<String, String> headers = clientResponse.getHeaders();
 
-        post.setHeader( HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_FORM_URLENCODED );
-
-        HttpResponse rsp = client.execute( host, post );
-
-        printResponse( rsp );
-
-        // should be an error, no content type was set
-        assertEquals( 200, rsp.getStatusLine().getStatusCode() );
-
-        Header[] headers = rsp.getHeaders( HttpHeaders.CONTENT_TYPE );
-
-        assertEquals( 1, headers.length );
-
-        assertEquals( MediaType.APPLICATION_JSON, headers[0].getValue() );
+        List<String> contentType = headers.get("Content-Type");
+        assertEquals(1, contentType.size());
+        assertEquals(MediaType.APPLICATION_JSON, contentType.get(0));
     }
 
 
-    /** Tests that application/x-www-url-form-encoded works correctly */
+    /**
+     * Tests that application/x-www-url-form-encoded works correctly
+     */
     @Test
     @Ignore("This will only pass in tomcat, and shouldn't pass in grizzly")
```

Comment:
We aren't using grizzly anymore ( at least I'm pretty sure we don't ). So this test should pass.